cmd/kubectl-package/export/push_test.go

```diff
@@ -1,65 +0,0 @@
-package export_test
-
-import (
-	"bytes"
-	"context"
-	"io"
-	"log"
-	"net/http"
-	"testing"
-
-	"github.com/google/go-containerregistry/pkg/crane"
-	"github.com/google/go-containerregistry/pkg/registry"
-	"github.com/google/go-containerregistry/pkg/v1/empty"
-	"github.com/stretchr/testify/assert"
-
-	"package-operator.run/package-operator/cmd/kubectl-package/export"
-)
-
-type writer struct {
-	resp *http.Response
-}
-
-func (w *writer) Header() http.Header { return w.resp.Header }
-
-func (w *writer) Write(data []byte) (int, error) {
-	buf := bytes.NewBuffer(data)
-	w.resp.Body = io.NopCloser(buf)
-
-	return len(data), nil
-}
-
-func (w *writer) WriteHeader(statusCode int) { w.resp.StatusCode = statusCode }
-
-type tripper struct {
-	handler http.Handler
-}
-
-func (t tripper) RoundTrip(req *http.Request) (*http.Response, error) {
-	if req.Body == nil {
-		req.Body = io.NopCloser(&bytes.Buffer{})
-	}
-	resp := &http.Response{Status: "ok", StatusCode: http.StatusOK, Header: http.Header{}, Request: req}
-	w := &writer{resp}
-	t.handler.ServeHTTP(w, req)
-
-	return resp, nil
-}
-
-func TestPush(t *testing.T) {
```

Comment:
Can we save this TestPush test and put it in `internal/packages/packageexport/image_test.go`

Reply:
Oops, yeah I messed that one up.